### ISSUE TITLE
cachy-browser: build fails on wasm C compiler wasi headers

### ISSUE THREAD

**Issue by jsmcconn** [CachyOS/CachyOS-PKGBUILDS#449]
Unable to build cachy-browser with makepkg.
```
git clone https://github.com/CachyOS/CachyOS-PKGBUILDS.git
cd CachyOS-PKGBUILDS/cachy-browser
makepkg -si
```

```
 0:06.37 Using wasi sysroot in /usr/share/wasi-sysroot
 0:06.37 checking for the wasm C compiler... /usr/bin/clang
 0:06.39 checking whether the wasm C compiler can be used... yes
 0:06.39 checking the wasm C compiler version... 18.1.8
 0:06.41 checking the wasm C compiler works... yes
 0:06.43 checking the wasm C compiler can find wasi headers...
 0:06.43 DEBUG: Creating `/tmp/conftest54xtt_fm.c` with content:
 0:06.43 DEBUG: | #include <string.h>
 0:06.43 DEBUG: | int
 0:06.43 DEBUG: | main(void)
 0:06.43 DEBUG: | {
 0:06.43 DEBUG: |
 0:06.43 DEBUG: |   ;
 0:06.43 DEBUG: |   return 0;
 0:06.43 DEBUG: | }
 0:06.43 DEBUG: Executing: `/usr/bin/clang -std=gnu99 --target=wasm32-wasi /tmp/conftest54xtt_fm.c --sysroot=/usr/share/wasi-sysroot -nodefaultlibs -lc -lwasi-emulated-process-clocks -lc++ -lc++abi %%PORTAGE_WORKDIR%%/wasi-sdk-%%WASI_SDK_VER%%-%%WASI_ARCH%%-linux/lib/clang/%%WASI_SDK_LLVM_VER%%/lib/wasi/libclang_rt.builtins-wasm32.a -c`
 0:06.43 DEBUG: The command returned non-zero exit status 1.
 0:06.43 DEBUG: Its error output was:
 0:06.43 DEBUG: | clang: error: no such file or directory: '%%PORTAGE_WORKDIR%%/wasi-sdk-%%WASI_SDK_VER%%-%%WASI_ARCH%%-linux/lib/clang/%%WASI_SDK_LLVM_VER%%/lib/wasi/libclang_rt.builtins-wasm32.a'
 0:06.43 DEBUG: | clang: warning: -lc: 'linker' input unused [-Wunused-command-line-argument]
 0:06.44 DEBUG: | clang: warning: -lwasi-emulated-process-clocks: 'linker' input unused [-Wunused-command-line-argument]
 0:06.44 DEBUG: | clang: warning: -lc++: 'linker' input unused [-Wunused-command-line-argument]
 0:06.44 DEBUG: | clang: warning: -lc++abi: 'linker' input unused [-Wunused-command-line-argument]
 0:06.44 ERROR: Cannot find wasi headers or problem with the wasm compiler. Please fix the problem. Or build with --without-wasm-sandboxed-libraries.
*** Fix above errors and then restart with "./mach build"
==> ERROR: A failure occurred in build().
    Aborting...
``` 

Was working as of 92c5a5ecaaddc439dde407cb2e5fea18c3d30455 .

**Comment by vnepogodin** [comment]
is that on archlinux/cachyos or you are trying to build it on gentoo(or just with portage)

**Comment by jsmcconn** [comment]
Building on up to date CachyOS install, see here for my temp fix just created:
https://github.com/jsmcconn/CachyOS-Browser-Common/commit/c19efd60b0863c04d35a7232ff5af9b642e6eb32

**Comment by jsmcconn** [comment]
This was fixed in 221f27b3f5552c572fc99293fcd51fbe88c1fc5b, closing issue.
